app/views/pages/staff.html.erb

```diff
@@ -89,7 +89,8 @@
           <div class="col s12 m6">
             <div class="card">
               <div class="card-content">
-                <span class="card-title"><a href="mailto:jay@operationcode.org">Jay Miller</a></span>
+                      <span class="card-title"><a href="mailto:jay@operationcode.org">Jay Miller</a>(<a href="https://twitter.com/kjaymiller">@kjaymiller</a>)</span>
```

Comment:
Looks like the indentation changed and there's an extra line. Do you mind cleaning that up?